packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -2548,16 +2567,39 @@ class TextSelectionGestureDetectorBuilder {
         _scrollPosition - _dragStartScrollOffset,
       );
       final Offset dragStartGlobalPosition = details.globalPosition - details.offsetFromOrigin;
+      final bool alternateBetweenEvenOdd = defaultTargetPlatform == TargetPlatform.windows;
```

Comment:
Since we have a specific behaviour for Windows, wondering if there should be a separate test also? Plus a comment on where this comes from maybe.

Reply:
Added